backend/experiment/models.py

```diff
@@ -45,30 +66,60 @@ def __str__(self):
     class Meta:
         verbose_name_plural = "Experiments"
 
-    def associated_blocks(self):
+    def associated_blocks(self) -> list[models.Model]:
+        """Return a list of all associated blocks for this experiment
+
+        Returns:
+            (list[Block]): Associated blocks
+        """
+
         phases = self.phases.all()
         return [block for phase in phases for block in list(phase.blocks.all())]
 
-    def export_sessions(self):
-        """export sessions for this experiment"""
+    def export_sessions(self) -> list[models.Model]:
+        """export sessions for this experiment
+
+        Returns:
+            (list[session.models.Session]): Associated sessions
+        """
+
         all_sessions = Session.objects.none()
         for block in self.associated_blocks():
             all_sessions |= Session.objects.filter(block=block).order_by("-started_at")
         return all_sessions
 
-    def current_participants(self):
-        """Get distinct list of participants"""
+    def current_participants(self) -> list[models.Model]:
+        """Get distinct list of participants
+
+        Returns:
+            (list[participant.models.Participant]): Associated participants
+        """
+
         participants = {}
         for session in self.export_sessions():
             participants[session.participant.id] = session.participant
         return participants.values()
 
-    def get_fallback_content(self):
-        """Get fallback content for the experiment"""
+    def get_fallback_content(self) -> list[models.Model]:
+        """Get fallback content for the experiment
+
+        Returns:
+            (list[ExperimentTranslatedContent]): Translated content
+        """
+
         return self.translated_content.order_by("index").first()
 
-    def get_translated_content(self, language: str, fallback: bool = True):
-        """Get content for a specific language"""
+    def get_translated_content(self, language: str, fallback: bool = True) -> list[models.Model]:
+        """Get content for a specific language
+        
+        Args:
+            language (str): Language code
+            fallback (bool): Return fallback language if language isn't available
+
+        Returns:
+            (list[ExperimentTranslatedContent]): Translated content
```

Comment:
I believe here we only return *one* `TranslatedContent` object. Or the function names are misleading.